cinema/serializers.py

```diff
@@ -1 +1,134 @@
-# write serializers here
+from rest_framework import serializers
+from cinema.models import (
+    Order,
+    Ticket,
+    CinemaHall,
+    Actor,
+    Genre,
+    Movie,
+    MovieSession
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("name", "rows", "seats_in_row")
+
+
+class CinemaHallDetailSerializer(serializers.ModelSerializer):
+    def get_capacity(self, obj):
```

Comment:
Type annotation is missing for the method `get_capacity`. According to the checklist, do not forget about type annotation in class methods.